pkg/policy/tests/102-pass.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  creationTimestamp: 2019-06-16T00:12:34Z
+  name: build-robot
```

Comment:
name: default

Reply:
decided to leave it as is, I think it's better to ensure that without the name the rule doesn't fail. If we set the name to `default` then we need to add the key with the right value to make it pass